### PR TITLE
Updated Cargo.toml to build using `cargo deb`

### PR DESCRIPTION
**Pull request by mynameisfiber**
I updated the `Cargo.toml` file and added a `package.metadata.deb`
subsection to define how to build a debian "deb" instal file using
`cargp deb`. This will allow debian/ubuntu users to install `alacritty`
using their systems package manager. It also will make it easier to
provide pre-built binaries for those systems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,16 +93,6 @@ xbps-install alacritty
...
-#### Ubuntu
-
-On Ubuntu, you need a few extra libraries to build Alacritty. Here's an `apt`
-command that should install all of them. If something is still found to be
-missing, please open an issue.
-
-```sh
-apt-get install cmake libfreetype6-dev libfontconfig1-dev xclip
-```
-
```
Comment (chrisduerr) [inline README.md]: These are still revelant for people trying to build alacritty on Ubuntu, I don't think removing this section is a good idea.

File: alacritty-completions.bash [hunk 2]
```diff
@@ -15,7 +15,6 @@ _alacritty()
 
     # If `--command` or `-e` is used, stop completing
     for i in "${!COMP_WORDS[@]}"; do
-        echo "${COMP_WORDS[i]}" >> ./testfile
```
Comment (chrisduerr) [inline alacritty-completions.bash]: Thanks a ton for catching this! I must have forgot removing it after testing the completions.

File: README.md [hunk 3]
```diff
@@ -50,6 +50,18 @@ cd alacritty-git
 makepkg -isr
 ```
 
+### Debian/Ubuntu
+
+Using `cargo deb` you can create and install a deb file.
```
Comment (chrisduerr) [inline README.md]: I'm not entirely sure about this and it's pretty much completely irrelevant and a super tiny nitpick, but I believe adding a `,` after `cargo deb` might be a good choice.

(Like this: "Using `cargo deb`, you can create and install a deb *(maybe .deb?)* file.")

File: alacritty-completions.bash (hunk 2 above)
Comment (mynameisfiber) [reply alacritty-completions.bash]: No worries!

**Comment by chrisduerr** [comment]
I definitely like the idea of adding a changelog, it's something that seems very useful to me. If you want to test out travis deployment, you can use travis on your own alacritty fork (that's usually how I test things). But if you don't have any experience with this, there's no rush to add this right now.

Taring up the manpages during the build process should be something we could do with travis (`tarup manpages && cargo deb` pretty much). So it seems like this PR should be good the way it is right now, thanks a lot!

**Pull request merged**